### PR TITLE
change documentation to use nvidia-docker

### PR DESCRIPTION
**Pull request by jendap**
Recommend nvidia-docker as the right way of running docker images.

### CODE REVIEW

**Comment by jendap** [comment]
FYI: @3XX0 @flx42 it is finally happening.

https://hub.docker.com/r/tensorflow/tensorflow/ is also updated.

**Comment by vrv** [comment]
Does this replace #2739 ?

**Comment by jendap** [comment]
I have not noticed the other one.

Maybe we should merge them. The #2739 has more comment. Here we have extra link to all tags  and deprecation warning in the old script.

Pick the one you like :-)

And we should merge it (or cherry pick it to 0.9).

**Comment by vrv** [comment]
That one has conflicts, so let's accept this one right now and we'll have the other author provide more as needed :)

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -358,16 +358,15 @@ The option `-p 8888:8888` is used to publish the Docker container᾿s internal p
...
-in the repo with these flags, so the command-line would look like
+For nvidia GPU support install [nvidia-docker](https://github.com/NVIDIA/nvidia-docker)
+and run
 
 ```bash
-$ path/to/repo/tensorflow/tools/docker/docker_run_gpu.sh gcr.io/tensorflow/tensorflow:gpu
+$ nvidia-docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow:gpu
 ```
 
+For more details see (tensorflow docker readme)[https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/docker].
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: s/tensorflow/TensorFlow

File: tensorflow/g3doc/get_started/os_setup.md [hunk 2]
```diff
@@ -358,16 +358,15 @@ The option `-p 8888:8888` is used to publish the Docker container᾿s internal p
 
 The format of the port mapping `hostPort:containerPort`. You can speficy any valid port number for the host port but has to be `8888` for the container port portion.
 
-If you're using a container with GPU support, some additional flags must be
-passed to expose the GPU device to the container. For the default config, we
-include a
-[script](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/docker_run_gpu.sh)
-in the repo with these flags, so the command-line would look like
+For nvidia GPU support install [nvidia-docker](https://github.com/NVIDIA/nvidia-docker)
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: NVidia

**Pull request merged**